experimental/dds/tree/src/EditLog.ts

```diff
@@ -534,8 +527,9 @@ export class EditLog<TChange = unknown> extends TypedEventEmitter<IEditLogEvents
 	}
 
 	private evictEdits(): void {
+		const minSequenceIndex = this.sequenceNumberToIndexMap.get(this._minSequenceNumber) ?? 0;
```

Comment:
It's possible that our current `_minSequenceNumber` is not in the map, but we should still pick the next lowest value.

[sequenceNumber, index, minSequenceNumber]
[0, 0, 0]
[1, 1, 1]
[3, 2, 2]

If we evict at this point, there is no entry in our map for the key "2", so we'll compute "0" for the `minSequenceIndex`. But we want to compute "1", because that's what the index was during op with sequence number 2; we know that because it's the value of the next lowest entry [1, 1]. We need to use a data structure that can query "next lower" (could use btree if you want, or cook up something more minimal). There should be a test for this case too.